Libraries/xcsdk/Sources/SDK/Platform.cpp

```diff
@@ -63,7 +64,7 @@ settings() const
 {
     std::vector<pbxsetting::Setting> settings = {
         pbxsetting::Setting::Create("PLATFORM_NAME", _name),
-        pbxsetting::Setting::Create("PLATFORM_DISPLAY_NAME", _description),
+        pbxsetting::Setting::Create("PLATFORM_DISPLAY_NAME", _description.value_or("")),
```

Comment:
What would happen if you just didn't include this particular setting if _description didn't exist?